package.json

```diff
@@ -142,10 +142,8 @@
       "@emotion/react@11.10.4": "patches/@emotion__react@11.10.4.patch",
       "@mui/base@5.0.0-alpha.100": "patches/@mui__base@5.0.0-alpha.100.patch",
       "react-devtools-inline@4.26.1": "patches/react-devtools-inline@4.26.1.patch",
-      "@protobufjs/inquire@1.1.0": "patches/@protobufjs__inquire@1.1.0.patch",
       "micromark@3.1.0": "patches/micromark@3.1.0.patch",
       "micromark-util-symbol@1.0.1": "patches/micromark-util-symbol@1.0.1.patch",
-      "@sentry/core@7.3.0": "patches/@sentry__core@7.3.0.patch",
```

Comment:
CC @Lanttcat

Reply:
I's OK. The root package.json have this dep.